### ISSUE TITLE
Print names of output files if it's not the name given for `--out`

### ISSUE THREAD

**Issue by ispringer** [quicktype/quicktype#1123]
```
$ quicktype --version
quicktype version 15.0.158
$ cat foo.ts
interface Tester {
    name: string;
}
$ quicktype foo.ts -o foo.java --debug
Error: Cannot read property 'split' of null.
$ echo $?
1
$ cat foo.java
cat: foo.java: No such file or directory
```

**Comment by schani** [comment]
The `--debug` option takes an argument, and we're not handling the case where it's missing.

Furthermore, Java will produce files named after the classes, so if you remove the `--debug` option you should end up with a `Tester.java` file.  Can you confirm that?

**Comment by ispringer** [comment]
Yes, Tester.java gets created. I didn't notice that before, because I was looking for foo.java. It would be helpful if it printed the name of the output file in the case where it is not the same as the filename specified via -o.

**Comment by schani** [comment]
Retested on current master (`node dist/index.js foo.ts -o foo.java --debug`). The crash is gone: instead of the unhelpful `Error: Cannot read property 'split' of null`, quicktype now reports `Option parsing failed: Missing value for command line option "debug"` and exits with status 1, since `--debug` requires a value that wasn't supplied. Dropping `--debug` still generates `Tester.java` correctly, as confirmed earlier in this thread. The request to print the actual output filenames when they differ from the `-o` argument is a separate enhancement that hasn't been implemented, so I'm closing this specific bug report; feel free to open a new issue for that feature request if it's still wanted. This fix will be included in the next release — if you can still reproduce the original crash on a current build, please reopen or comment with details.
